### PR TITLE
Provides default joint angles & nullspace bias to Cartesian script

### PR DESCRIPTION
**Pull request by ShaneRozenLevy**
- modifies go_to_cartesian_pose.py
- Now provides current joint configuration as nullspace joint bias.
- Adds comments to clean up code

### CODE REVIEW

File: intera_examples/scripts/go_to_cartesian_pose.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ def main():
     -> Jog arm with Relative Pose (in tip frame)
     -> x=0.01, y=0.02, z=0.03 meters, roll=0.1, pitch=0.2, yaw=0.3 radians
     -> The fixed position and orientation paramters will be ignored if provided
-
     """
+
```
Comment (IanTheEngineer) [inline intera_examples/scripts/go_to_cartesian_pose.py]: Please remove the extra white space lines

File: intera_examples/scripts/go_to_cartesian_pose.py [hunk 2]
```diff
@@ -158,7 +159,15 @@ def main():
                     pose.orientation.w = args.orientation[3]
             poseStamped = PoseStamped()
             poseStamped.pose = pose
-            waypoint.set_cartesian_pose(poseStamped, args.tip_name, args.joint_angles)
+
+            if not args.joint_angles:
```
Comment (IanTheEngineer) [inline intera_examples/scripts/go_to_cartesian_pose.py]: This indentation seems aggressive

**Comment by ShaneRozenLevy** [comment]
A recent change to the motion controller requires the nullspace joint bias to be provided with each waypoint. The current version of go_to_cartesian_pose.py does not provide the nullspace joint bias unless the joint angles are provided as an argument. 
`$ rosrun intera_examples go_to_cartesian_pose.py -p 0.4 -0.3 0.18 -o 0.0 1.0 0.0 0.0 -t right_hand`
Should replicate the bug, resulting in
`[ERROR] [1528747426.219658]: Motion controller failed to complete the trajectory with error CARTESIAN_INTERPOLATION_FAILED`
being published to the terminal.

**Comment by IanTheEngineer** [comment]
Excellent. Thanks!

**Pull request merged**